src/documents/filters.py

```diff
@@ -417,6 +417,13 @@ def _parse_atom(self, id_or_name, op, value) -> Q:
         value_field_name = CustomFieldInstance.get_value_field_name(
             custom_field.data_type,
         )
+        if custom_field.data_type == CustomField.FieldDataType.MONETARY and op in (
+            "gt",
+            "gte",
+            "lt",
+            "lte",
+        ):
+            value_field_name = "value_monetary_amount"
```

Comment:
Is there any reason we don't do `on in self.EXPR_BY_CATEGORY["arithmetic"]` here? I guess we should also handle "range" ?

Reply:
Sure I can change that. Range can be for another day, its not really handled in the UI at the moment.

Reply:
Yeah I guess it's more for being consistent with this ("range" is marked as supported but not handled correctly), in case folks are developing their own API consumer. https://github.com/paperless-ngx/paperless-ngx/blob/6ee2d1ac7dbfea8b274604333df7b9f2335ce97a/src/documents/filters.py#L264

But indeed this is pretty cool.